src/data_info/services/sharing.clj

```diff
@@ -148,7 +148,7 @@
 (defn anon-file-url
   [p]
   (let [aurl (url/url (cfg/anon-files-base))]
-    (str (-> aurl (assoc :path (ft/path-join (:path aurl) (string/replace p #"^\/" "")))))))
+    (str (-> aurl (assoc :path (apply ft/path-join (:path aurl) (map url/url-encode (string/split (string/replace p #"^\/" "") #"\/"))))))))
```

Comment:
Oh, hah, I didn't even think about it. I just copied what was in the other regex. I'll change it, may as well prevent the same thing happening again :)